src/surface.lisp

```diff
@@ -17,8 +17,11 @@
   (c-ref surface sdl2-ffi:sdl-surface :pitch))
 
 (defun surface-format-format (surface)
-  (enum-key '(:enum (sdl-pixel-format))
-            (c-ref surface sdl2-ffi:sdl-surface :format :format)))
+  (c-ref (c-ref surface
+                sdl2-ffi:sdl-surface
+                :format *)
+         sdl2-ffi:sdl-pixel-format
+         :format))
 
```

Comment:
Er, why was this changed?  You should not need nested `C-REF`, and this should return the *keyword* not just the numeric value.